### ISSUE TITLE
repush to directory

### ISSUE THREAD

**Issue by florian-wagner** [reHackable/scripts#3]
Hi @Ameb,

thanks for the scripts. I like `repush` as it uses the web interface and does not require a reboot (as pdf2remarkable.sh). It works fine, however, I did'n have success to push to a directory.

``` bash
Successfully established connection, please do not lock your device until the script has completed!

===================================================================================================================================
Shipping documents to output directory. It is highly recommended to refrain from using your device until this script has completed!
===================================================================================================================================

Temporarily disabling Wi-Fi to prevent conflicts with the cloud

Shipping 'test.pdf'
Failed to access remote metdata for '_tmp_repush_test_tmp_repush_'
Applying changes...
Re-enabling Wi-Fi in 5 seconds...

Exit request sent.
Successfully transferred  out of 1 documents
```
The command was `repush -d Privat test.pdf`, where `Privat` is the visible name of an existing folder on the rM. Any clues?

**Comment by Ameb** [comment]
Everything was made by @CTXz :). I just used some duct tape to fix some edge cases. I can't check or try to fix this right now, but i think it won't be easy since right now the beta interface is bugged and always uploads the documents to the root folder iirc. Same for the desktop apps. 
So the duct tape fix here would be to upload the document and then use the web api to move it.

**Comment by CTXz** [comment]
### Oops

This issue is present as of remarkable software codex version `1.2.0.344`. It has been fixed by [this commit](https://github.com/reHackable/scripts/commit/a002bad6a4439ae971c8b667c646bb046ef75f03) , however, I seem to have forgotten to push the change :sweat:. Apologies!

As of codex version `1.2.0.344`, file extensions are no longer trimmed, but instead, included in the file name. Upon specification of an output directory,  repush copies the file(s) to /tmp/ and gives it/them a name by the following syntax: `_tmp_repush_<FILENAME>_tmp_repush.<EXTENSION>`. On the previous software version, the visible name would trim the extension, so the file on the device would be named `_tmp_repush_<FILENAME>_tmp_repush`. From there on, repush would search for a file that went by the name of `_tmp_repush_<FILENAME>_tmp_repush` and move it into the correct directory. Finally it'd rename it back to its original filename and restart xochitl.

After the update however, the files maintain their extension. So when repush searches for `_tmp_repush_<FILENAME>_tmp_repush`, it won't find the file. Instead, it must search for `_tmp_repush_<FILENAME>_tmp_repush.<EXTENSION>`, which has been corrected in a002bad6a4439ae971c8b667c646bb046ef75f03

**To fix this issue, simply pull the master branch and try again and tells us how it went!**

**Comment by florian-wagner** [comment]
@CTXz Thanks for a lot the quick update! Now it works as expected.
